app/views/tweets/tweets.erb

```diff
@@ -0,0 +1,24 @@
+<% if flash.has?(:message) %>
+  <%= flash[:message] %>
+<% end %>
+
+<h1>Welcome, <%= @user.username %>! </h1>
+
+
+<h4><a href="/users/<%=@user.slug%>">Your Homepage</a></h4>
+
+<h4><a href="/tweets/new">Make a Tweet</a></h4>
+
+
+<br>
+
+<h2>All Tweets - Click to go to edit page</h2>
+  <ul><% Tweet.all.each do |tweet| %>
```

Comment:
In the MVC design pattern we should avoid database calls in the view. You would be better served setting this to an instance variable in the `get '/tweets' do` method

```ruby
@tweets = Tweet.all
```